settings.gradle.kts

```diff
@@ -56,6 +56,10 @@ pluginManagement {
   }
 }
 
+plugins {
+  id("org.gradle.toolchains.foojay-resolver-convention") version("0.8.0")
+}
```

Comment:
Can we move toolchain stuff out of this PR?

BTW: we have an existing PR to enable Java toolchains here, although with some issues: https://github.com/apple/pkl/pull/262